pages/Jetpacks.md

```diff
@@ -0,0 +1,7 @@
+The Electric Jetpack is a useful [Technical Gadget](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Technical-Gadget) which allows you to fly through the World.
+
+## Obtaining
+The Electric Jetpacks can be crafted in an [Enhanced Crafting Table](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+Just Charge it put it on and press shift and you will fly into the Sky!
```

Comment:
Also maybe say "crouch" instead of "press shift", since you could bind that to any otherr key than shift.